src/components/editions/headerImageCaption.tsx

```diff
@@ -46,14 +46,6 @@ const HeaderImageCaptionStyles = (
 		color: ${neutral[100]};
 		${textSans.small()};
 		box-sizing: border-box;
-
-		${from.tablet} {
-			padding-left: ${remSpace[6]};
-		}
-
-		${from.desktop} {
-			padding-left: 9rem;
-		}
```

Comment:
@joecowton1 do you know why this was here in the first place? maybe there's a good reason for it and I shouldn't remove it 🤔

Reply:
@faresite it was to keep the caption inline with the left margin on full screen images, but i get the feeling it shouldn't apply to all images, just full screen ones. Does that make sense?